.github/workflows/test.yml

```diff
@@ -9,13 +9,18 @@ on:
 
 jobs:
 
+
   test:
     strategy:
       fail-fast: false
       max-parallel: 3
       matrix:
-        os: [ ubuntu, windows, macos ]
+        include:
```

Comment:
These made me smile :-)

Reply:
I like them a lot :D
They feel stupid at first, but they are quite useful as the complexity is increased: https://github.com/ghdl/ghdl/actions/runs/856883375. Yet, I understand if you don't want those characters in the sources :wink: